### PR TITLE
Fixed missing links

### PR DESCRIPTION
**Pull request by srk97**
Link to JS-LoggedInAsync could not be found

### CODE REVIEW

File: src/pages/miscellaneous/guides-to-back-end-projects/index.md [hunk 1]
```diff
@@ -39,4 +39,4 @@ Here you will find articles that will help you learn more and complete your back
 <a>highlighted.items/are/hypothetical</a> URLs that allow access to different pages in your app  
 _italicizedItems_ are function and object names within your code
 
-<a>PREVIOUS</a> <a href='http://forum.freecodecamp.com/t/front-end-file-structure/14266' target='_blank' rel='nofollow'>NEXT</a>
\ No newline at end of file
+<a>PREVIOUS</a> <a href='http://forum.freecodecamp.com/t/front-end-file-structure/14266' target='_blank' rel='nofollow'>NEXT</a>
```
Comment (adiparab14) [inline src/pages/miscellaneous/guides-to-back-end-projects/index.md]: @srk97 Are these navigation at the end required?

Comment (srk97) [reply src/pages/miscellaneous/guides-to-back-end-projects/index.md]: The "PREVIOUS" button is unnecessary since it's a dead end.

**Comment by srk97** [comment]
@adiparab14 I've removed the links at the footer

**Comment by johnkennedy9147** [comment]
@QuincyLarson Hi, this article looks like something pulled out of forum. Do you think it serves a useful purpose in Guide? Should we just remove the article entirely?

**Comment by QuincyLarson** [comment]
@johnkennedy9147 This seems helpful in the sense that it's an Angular-focused approach to the backend, but it isn't named as such. I'm going to accept this, then open a follow-up PR renaming the guide.

**Pull request merged**